Filed/"All Available Information Concerning That Flight".md

```diff
@@ -0,0 +1,28 @@
+[[FAR 91.103]] uses this phrase and it's important for the student to understand what it means, and how to get the information it requires.  The acronym students use to remember this is [[NWKRAFT]].
```

Comment:
Blocked the merge on this one, since I think it'll cause unforeseen heartache for others down the road.

Rest of the PR I have a few noted as "consider" and "maybe", up to you on those. I often put things in footnotes ("^[]") on Obsidian if they seem useful to know or link, but not worth putting in main text.

Reply:
Yes, makes sense.  I'll remove the quotes.